apps/src/templates/sectionAssessments/SectionAssessments.jsx

```diff
@@ -64,9 +65,8 @@ class SectionAssessments extends Component {
             onChange={this.props.setAssessmentId}
           />
         </div>
-        <div>
           <MultipleChoiceByStudentContainer />
-        </div>
```

Comment:
It looks like you removed the closing div, but forgot to remove the opening one.

Reply:
Got it! Thanks